src/main/java/hudson/plugins/ec2/EC2AbstractSlave.java

```diff
@@ -199,6 +199,8 @@ public EC2Cloud getCloud() {
             return 7;
         case C4Large:
             return 7;
+        case C5Large:
+            return 7;
```

Comment:
What data source is used to infer these numbers?

Reply:
I feel like using the AWS Pricing files might be a good method for determining this. One question in my mind is if these values are hard-coded (as they are now) or if a "pricing" API call (https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/pricing/model/package-summary.html) is used to determine the "vcpu" or "ecu" value (and hence, the NumExecutors).

Reply:
Make it simple - no API call.

Reply:
@colinbjohnson I would welcome a contribution that uses the API call rather than this hardcoded nonsense that has to be updated all the time.